importer/helper.go

```diff
@@ -257,3 +244,38 @@ func (paths *blockTarPaths) Swap(i, j int) {
 	(*paths)[i] = (*paths)[j]
 	(*paths)[j] = t
 }
+
+// GetReader creates a genesis reader given the inputs
+func GetReader(genesisJSONPath string, client *algod.Client, l *log.Logger) io.Reader {
```

Comment:
Nice pattern! The only suggestion is to include what is being read somehow, like "GetGenesisFile". A `type GenesisFileReader io.Reader` could also help clarify what is going on and help link this function to InitialImport.